.code-samples.meilisearch.yaml

```diff
@@ -190,6 +190,12 @@ update_displayed_attributes_1: |-
   ])
 reset_displayed_attributes_1: |-
   client.index('movies').resetDisplayedAttributes()
+get_faceting_1: |-
+  client.index('books').getFaceting()
+update_faceting_1: |-
+  client.index('books').updateFaceting({ maxValuesPerFacet: 2 })
+reset_faceting_1: |-
+  client.index('books').resetFaceting()
```

Comment:
I'm not sure where these code samples come from, if they come from an issue can you share it?
They are not correct :( 
```suggestion
```

They should be added in the following:
- line 560
- line 563
- Needs that one as well: `getting_started_faceting` based on [this](https://github.com/meilisearch/documentation/blob/c7bcbac3f7d01f953c8087d4a3f5d7c62f785eed/.code-samples.meilisearch.yaml#L975)

Reply:
I got the link from the Rust issue: https://github.com/meilisearch/integration-guides/issues/208.

Reply:
Thanks I will update it 🙏

Reply:
@bidoubiwa, like this? https://github.com/meilisearch/meilisearch-js/pull/1344/commits/e0eeefc2f8da15ad26e699ff510d50558dfa0ff8

Reply:
Perfect!